### PR TITLE
Add Logging in the Renderer

### PR DESCRIPTION
**Pull request by cwfitzgerald**
There was too little logging in the renderer, so I added some.

### CODE REVIEW

File: bve-render/src/render.rs [hunk 1]
```diff
@@ -130,6 +130,7 @@ pub fn create_pipeline(
     fs: &ShaderModule,
     samples: MSAASetting,
 ) -> RenderPipeline {
+    debug!("Creating opaque pipeline: samples: {:?}", samples);
```
Comment (DethRaid) [inline bve-render/src/render.rs:133]: Protip: log the name of the pipeline you're creating

File: bve-render/src/render.rs [hunk 2]
```diff
@@ -203,6 +208,10 @@ pub fn create_depth_buffer(device: &Device, size: PhysicalSize<u32>, samples: MS
 }
 
 pub fn create_framebuffer(device: &Device, size: PhysicalSize<u32>, samples: MSAASetting) -> TextureView {
+    debug!(
```
Comment (DethRaid) [inline bve-render/src/render.rs:211]: This doesn't matter yet, but you probably will want to name your framebuffers and log their names here

File: bve-render/src/render.rs (hunk 1 above)
Comment (cwfitzgerald) [reply bve-render/src/render.rs:133]: I did, `opaque`

File: bve-render/src/render.rs (hunk 2 above)
Comment (cwfitzgerald) [reply bve-render/src/render.rs:211]: I mean I only got one.

Comment (DethRaid) [reply bve-render/src/render.rs:211]: For now, but once you get more complex you may have lots of framebuffers. Just a thought

File: bve-render/src/render.rs (hunk 1 above)
Comment (DethRaid) [reply bve-render/src/render.rs:133]: oh

File: bve-render/src/render.rs (hunk 2 above)
Comment (cwfitzgerald) [reply bve-render/src/render.rs:220]: Yeah I should be specific with my non-main framebuffer

**Pull request merged**